cli/node.rs

```diff
@@ -35,6 +35,15 @@ pub fn resolve_specifier_into_node_modules(
     .unwrap_or_else(|| specifier.clone())
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub enum CliCjsAnalysis {
+  Esm,
+  Cjs {
+    exports: Vec<String>,
+    reexports: Vec<String>,
+  },
+}
```

Comment:
Can you add some docstrings to explain these variants? `CliCjsAnalysis::Esm` is confusing at first glance.

Reply:
Added.